### DISCUSSION TITLE
[Feature] Drag to select more photos on mobile

### DISCUSSION THREAD

**Discussion by Roy416** [immich-app/immich#3999] [Feature Request]
### The feature

Currently you hold on a photo, the selection UI appears. You would then be able to select multiple photos by dragging your finger over other unselected photos. Making it easier to quickly select a few photos to delete or add to albums. This feature is available on google photos and apple's photo app if you need a demo.

Not sure if possible on Web but would be cool.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by maxshcherbina** [comment]
This is one of the features I'm missing on the Immich app the most. I would really love to see this! 
I pick which photos to share with my partner in our shared album, and it can be an annoying experience tapping away at photos. Yes, I can select the whole day first and then tap the photos I don't want to include and that saves time, but I'd love the option to swipe as well! ❤️

**Comment by namu-lee** [comment]
I really want this feature too. Selecting many pictures taken on different day to share is very hard right now.

**Comment by daschillinger** [comment]
I hope it will come, too.

**Comment by daschillinger** [comment]
Maybe this could help? It's not the same but almost similar behavior. But I don't know if it is a tableview
https://developer.apple.com/documentation/uikit/uitableviewdelegate/selecting_multiple_items_with_a_two-finger_pan_gesture
